index.js

```diff
@@ -552,10 +552,14 @@ client.on('guildMemberAdd', async member => {
         }
         const channel = client.channels.cache.get(joinChannel)
         if(channel) {
-            const mes = joinMessage.split('{[USER]}').join(`<@${member.id}>`).split('{[SERVER]}').join(`${member.guild.name}`)
-            channel.send(mes).catch(err => {
-                console.log(err)
-            })
+	const mes = joinMessage
+	const mssembed = new Discord.MessageEmbed()
+	.setTitle(`Weclome to ${member.guild.name}`)
+	.setColor('#000000')
+	.setThumbnail(member.guild.iconURL())
+        .setDescription(`Greetings <@${member.user.id}>, ${mes}`)
```

Comment:
The description shouldn't include ```Greetings <@${member.user.id}>, ```

Also mes should be ```joinMessage.split('{[USER]}').join(`<@${member.id}>`).split('{[SERVER]}').join(`${member.guild.name}`)``` rather than just joinMessage

That way the whole thing is configurable easily